sections/docker/smaller_base_images.spanish.md

```diff
@@ -0,0 +1,12 @@
+
+Imágenes grandes de Docker pueden generar una mayor exposición a vulnerabilidades y a un mayor consumo de recursos. A menudo, no es necesario instalar ciertos paquetes en tiempo de ejecución que son necesarios para la compilación.
+Extraer y almacenar imágenes más grandes será más costoso a escala, cuando se trata de imágenes más grandes. Por diseño, es posible que las imágenes pequeñas no vengan con las librerías comunes necesarias para compilar módulos nativos o paquetes útiles para depurar (por ejemplo, curl) preinstaladas.
+El uso de las variantes de imágenes de Alpine Linux puede reducir la huella en términos de recursos utilizados y la cantidad de vectores de ataque presentes en sistemas con todas las funciones. La imagen de Docker de Node.js v14.4.0 tiene un tamaño de ~345MB frente a ~39MB de la versión Alpine, que es casi 10 veces más pequeña.
+Una variante delgada basada en Debian, que tiene solo 38MB de tamaño y contiene los paquetes mínimos necesarios para ejecutar Node.js, también es una excelente opción.
+
+### Cita de Blog: "Si desea reducir sus imágenes de Docker, hacer que sus servicios se inicien más rápido y sean más seguros, entonces pruebe Alpine"
+
```

Comment:
"hacer" suena condescendiente, queda mejor "haga"

Reply:
Creo que queda bien 'hacer' por cómo está planteada la cita.

'If you want to shrink your Docker images, have your services start faster and be more secure then try Alpine out.'

Yo la interpreto como 'Si quieres hacer esto, hacer esto otro y también esto otro, entonces prueba esto.'. ¿Qué opinas?